labs/002-install-lttng-on-ubuntu/README.md

```diff
@@ -8,28 +8,22 @@ In this lab, you will install lttng on an Ubuntu machine
 
 ### Task 1: Install lttng for kernel tracing
 
-First install the lttng PPA repository:
+First make sure your package manager list is up to date
```

Comment:
I prefer that we ask the reader to install the correct PPA as apt default packages may not have the correct versions.
For example, I just checked and the lttng-modules-dkms version does not match the lttng-tools version.

Reply:
OK - in that case we probably just need to point to a different PPA because the PPA in the current doc is for <= Ubuntu 18.04.

I guess updating the PPA to https://launchpad.net/~lttng/+archive/ubuntu/stable-2.13 makes the most sense then? 

I'll provide an update assuming that is correct.

Reply:
Yes that would be perfect, thank you !